src/app/about/TrackRecord.tsx

```diff
@@ -0,0 +1,30 @@
+export default function TrackRecord() {
+  return ( 
+    <div className="container my-8">
+        <div className="xl:grid xl:grid-cols-5 lg:grid lg:grid-cols-4 p-4 gap-6">
+
+            <div className="xl:col-span-2 py-4">
+            <h2 className="text-5xl font-semibold">Our Track Record Speaks for Itself</h2>            
```

Comment:
super nitpick but this looks compressed when it runs over 2 lines, in the figma its line-height 130% but I think either leading-1.3 or leading-tight works